ArmRobot/Assets/Scripts/Camera/RandomizerPositionCamera.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomizerPositionCamera : MonoBehaviour
+{
+    public float changePositionCamera = 0.005f;
+    public float initialCameraPositionXAxis = 0.0f;
+    public float initialCameraPositionYAxis = 1.8f;
+    public float initialCameraPositionZAxis = 0.0f;
+    
+    public float rotationCamera = 1.0f;
+    public float initialCameraRotationXAxis = 90.0f;
+    public float initialCameraRotationYAxis = 0.0f;
+    public float initialCameraRotationZAxis = 0.0f;
```

Comment:
Use a `Vector3` instead of three separate x, y, z variables.

Reply:
Also, set the initialCameraRotation from the current rotation in the `Start()` method, so that these hardcoded values are not necessary.